app/views/LoginView/index.tsx

```diff
@@ -23,7 +24,9 @@ const LoginView = () => {
 	useLayoutEffect(() => {
 		navigation.setOptions({
 			title: title ?? 'Rocket.Chat',
-			headerRight: () => <HeaderButton.Legal testID='login-view-more' navigation={navigation} />
+			headerRight: () => (
+				<HeaderButton.Legal accessibilityLabel={i18n.t('More')} testID='login-view-more' navigation={navigation} />
```

Comment:
`HeaderButton.Legal` is always going to have the same icon and we'll always going to call it `More`.
This change should be applied on the component instead of as a prop.
The same way you did on `HeaderButton.Drawer`.